src/ring_theory/valuation/basic.lean

```diff
@@ -492,6 +492,12 @@ variables {h0} {h1} {hadd} {hmul} {r : R}
 @[simp]
 theorem of_apply : (of f h0 h1 hadd hmul) r = f r := rfl
 
+/-- The `valuation` associated to an `add_valuation` (useful if the latter is constructed using
+`add_valuation.of`). -/
+def valuation : valuation R (multiplicative (order_dual Γ₀)) := v
```

Comment:
Does `simps` produce the lemma automatically?

Reply:
`@[simps]` gives me the error message `Invalid `simps` attribute. The body is not a constructor application:
  v`

Reply:
:( for `simps`, but thanks!

bors d+